sdk/pchain/send_transaction.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package pchain
+
+import (
+	"time"
+
+	"github.com/ava-labs/avalanche-cli/sdk/multisig"
+	"github.com/ava-labs/avalanche-cli/sdk/utils"
+	"github.com/ava-labs/avalanche-cli/sdk/wallet"
+	"github.com/ava-labs/avalanchego/wallet/subnet/primary/common"
+)
+
+func SendTransaction(
```

Comment:
Why don't we just move sending transaction to wallet like https://github.com/ava-labs/avalanche-tooling-sdk-go/blob/fef92d1127dc37da50a9307a78663b81a04b971e/wallet/wallet.go#L99?

Reply:
I have not issues in moving this into wallet, but I have a slight preference for non OOP style, which is also the TS SDK style

Reply:
anyway, we can have both options (if we find out the TS SDK provides both options)